docs/platform/reference/eol-for-major-versions.md

```diff
@@ -5,25 +5,72 @@ sidebar_label: Service & tool lifecycle
 
 End of life (EOL) is the date after which Aiven services and tools are no longer supported or maintained.
 
-## Aiven services
+## Aiven version support and upstream EOL
 
 Aiven aims to follow the EOL schedule set by the original authors and
 maintainers of the open source software (the upstream projects). Once
 the upstream project retires a specific version, they do not receive
 security updates and critical bug fixes anymore by the maintainers.
 
-Outdated services don't offer the level of protection our customers
+Outdated services don't offer the level of protection you
 need, so Aiven follows the upstream project's EOL schedule to ensure
 that Aiven services are always running on supported versions.
 
-### Version numbering
+## Service version numbering
 
 Aiven services inherit the upstream project's software versioning
 scheme. Depending on the service, a major version can be either a single
 digit (for example, PostgreSQL® 14) or in the format `major.minor` (for
 example, Kafka® 3.2). The exact version of the service is visible in the
 [Aiven Console](https://console.aiven.io/) when the service is running.
 
+## Multi-versioned vs single-versioned services
+
+There are two types of Aiven services with respect to versions:
+
+-   [Multi-versioned services](/docs/platform/reference/eol-for-major-versions#aiven-multi-versioned-services-eol):
+    You can choose the version for your service.
+-   [Single-versioned services](/docs/platform/reference/eol-for-major-versions#aiven-single-versioned-services-eol):
+    Only one default version is available at a time.
+
+## EOL policy for major versions
+
+The Aiven EOL policy applies only to services where you control the major version. It
```

Comment:
This sentence still feels a bit vague (where you control the major version.). Suggest clarifying the scope like:

> Aiven EOL policy applies only to services where you select the major version (for example, PostgreSQL or Apache Kafka). Services that run a single version managed by Aiven are not included. This policy covers both running and powered-off services on the affected version.